### DISCUSSION TITLE
Spinner hud rotates when `barrel roll` mod is enabled.

### DISCUSSION THREAD

**Discussion by GabuTheDev** [ppy/osu#28467] [Q&A]
### Type

Cosmetic

### Bug description

When using the `barrel roll` mod, the spinner skin sprites along with the spinner bonus and rpm counter ui elements are being rotated too, which is a weird and unexpected behaviour.

### Screenshots or videos

(can't upload directly to github, my video is above the 10MB limit)
[video here](https://drive.google.com/file/d/1XlZnldz6ahP_JRdV6HbKIVr6gOUjPMFc/view?usp=sharing)

### Version

2024.521.2-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/15809645/compressed-logs.zip)

**Comment by bdach** [comment]
Dunno if I'd consider this a bug to be honest.

**Reply by smoogipoo** [reply]
Let's keep this as a discussion for now.

**Reply by GabuTheDev** [reply]
Returning to this. I still don't think this is the intended behaviour, especially when talking about the RPM counter. How are you supposed to read numbers that constantly rotate? For the skin sprites, maybe it makes sense considering the nature of the implementation, but it's clearly not what a user would expect.

**Comment by DGCK81LNN** [comment]
the bounding boxes of playfield elements in the skin editor are not rotated correctly. An edge case tho

![osu_2024-08-08_13-55-14](https://github.com/user-attachments/assets/4b09fc9d-3c32-4342-af7e-92f0aac0d8c3)
